docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "How-To: Load Data in ML.NET"
+description: Load data file and streaming data into ML.NET
+ms.date: 04/03/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to load data from file and streaming data sources.
+---
+
+# How-To: Learn how to load data from file and real-time sources in ML.NET
+
+This how-to shows you how to load data for processing and training into ML.NET that's stored in files or real-time / streaming data sources.
```

Comment:
this it'd read better if:

This how-to shows you how to load data for processing and training into ML.NET. The data is originally stored in files or real-time / streaming data sources